src/app/admin/packingSlotsTable/PackingSlotsTable.tsx

```diff
@@ -326,26 +337,60 @@ const PackingSlotsTable: React.FC = () => {
         <>
             {errorMessage && <ErrorSecondaryText>{errorMessage}</ErrorSecondaryText>}
             {rows && (
-                <DataGrid
-                    rows={rows}
-                    columns={packingSlotsColumns}
-                    editMode="row"
-                    rowModesModel={rowModesModel}
-                    onRowModesModelChange={setRowModesModel}
-                    onRowEditStop={handleRowEditStop}
-                    processRowUpdate={processRowUpdate}
-                    slots={{
-                        toolbar: EditToolbar,
-                        loadingOverlay: LinearProgress,
-                    }}
-                    slotProps={{
-                        toolbar: { setRows, setRowModesModel, rows },
-                    }}
-                    loading={isLoading}
-                />
+                <DataGridStyling>
+                    <DataGrid
+                        rows={rows}
+                        columns={packingSlotsColumns}
+                        editMode="row"
+                        rowModesModel={rowModesModel}
+                        onRowModesModelChange={setRowModesModel}
+                        onRowEditStop={handleRowEditStop}
+                        processRowUpdate={processRowUpdate}
+                        slots={{
+                            toolbar: EditToolbar,
+                            loadingOverlay: LinearProgress,
+                        }}
+                        slotProps={{
+                            toolbar: { setRows, setRowModesModel, rows },
+                        }}
+                        loading={isLoading}
+                        getRowClassName={(params) =>
+                            (params.indexRelativeToCurrentPage + 1) % 2 === 0 ? "even" : "odd"
```

Comment:
Shall we have a class name that's more specific? Maybe datagrid-row-even etc? I think even and odd are _probably_ good enough but feel like we could be a bit more specific